test/e2e-common.sh

```diff
@@ -191,7 +191,8 @@ function install_istio() {
   fi
 
   local NET_ISTIO_DIR=$(mktemp -d)
-  git clone --quiet --depth 1 --branch master https://github.com/knative-sandbox/net-istio.git ${NET_ISTIO_DIR}
+  git clone --quiet --branch master https://github.com/knative-sandbox/net-istio.git ${NET_ISTIO_DIR}
+  git --git-dir "${NET_ISTIO_DIR}/.git" checkout 7398a24a5fa9b06154ce5ef6b551fce856591b08
```

Comment:
This is a band-aid.  It would be good to understand what our strategy is going to be for this moving forward.

Reply:
One thing we can do is to include the commit ID in the `net-istio.yaml`. It is produced by `hack/release.sh` so we have complete control about what information we put in there.